src/ExpressServer.js

```diff
@@ -1,17 +1,19 @@
-const express = require('express');
-const cors = require('cors');
-const boredApiRouteHandler=require('./Routes/ExperimentOne/BoredApiCaller')
+const express = require("express");
+const cors = require("cors");
+const boredApiRouteHandler = require("./Routes/ExperimentOne/BoredApiCaller");
+const tmdbApiRouteHandler = require("./Routes/ExperimentTwo/TMDBApiCaller");
 
 const app = express();
 
 //MiddleWare Setup
-app.use(cors());
+app.use(cors()); // Is this required to get the Keys etc
 
 //Route the Requests
-app.get('/boredApi',boredApiRouteHandler);
+app.get("/boredApi", boredApiRouteHandler);
+app.get("/tmdbApiTrending", tmdbApiRouteHandler);
 
 //Start the server
-const port=process.env.PORT || 5000
-app.listen(port,()=>{
-   console.log('Server has started on port ' + port)
-});
\ No newline at end of file
+const port = process.env.PORT || 5000;
+app.listen(port, () => {
+  console.log("Server has started on port " + port);
```

Comment:
my personal opinion but i'd use template string here
```suggestion
  console.log(`Server has started on port ${port}`);
```

Reply:
Yeah I agree